src/coreclr/CMakePresets.json

```
@@ -30,6 +30,22 @@
         }
       }
     },
+    {
+      "name": "osx-base",
+      "hidden": true,
+      "inherits": "base",
+      "cacheVariables": {
+        "CLR_CMAKE_TARGET_OS": "darwin"
+      },
+      "vendor": {
+        "microsoft.com/VisualStudioSettings/CMake/1.0": {
+          "hostOS": [
+            "macOS",
+            "OSX"
```

According to documentation and IntelliSense, only `macOS` is supported in this field.

```suggestion
            "macOS"
```

---

Thank you for pointing out.